sphere-models/src/it/java/io/sphere/sdk/products/ProductProjectionSearchFilterTypesIntegrationTest.java

```diff
@@ -158,54 +210,121 @@ public void filtersLocEnumLabelAttributes() throws Exception {
     }
 
     @Test
-    public void filtersNumberAttributes() throws Exception {
+    public void locEnumLabelAttributesFacets() throws Exception {
+        final TermFacetExpression<ProductProjection, String> facetExpr = model().allVariants().attribute().ofLocalizableEnum(ATTR_NAME_LOC_ENUM).label().locale(GERMAN)
+                .faceted().byAllTerms();
+        final ProductProjectionSearch search = ProductProjectionSearch.ofStaged().withFacets(facetExpr);
+        assertThat(executeAndReturnTerms(search, facetExpr)).containsOnlyElementsOf(asList(TermStats.of("zwei", 1), TermStats.of("drei", 1)));
+    }
+
+    @Test
+    public void numberAttributesFilters() throws Exception {
         final ProductProjectionSearch termSearch = ProductProjectionSearch.ofStaged()
                 .withQueryFilters(model -> model.allVariants().attribute().ofNumber(ATTR_NAME_NUMBER)
                         .filtered().by(valueOf(5D)));
         assertThat(executeAndReturnIds(termSearch)).containsOnly(product1.getId());
+
         final ProductProjectionSearch rangeSearch = ProductProjectionSearch.ofStaged()
                 .withQueryFilters(model -> model.allVariants().attribute().ofNumber(ATTR_NAME_NUMBER)
                         .filtered().byLessThanOrEqualTo(valueOf(5D)));
         assertThat(executeAndReturnIds(rangeSearch)).containsOnly(product1.getId());
     }
 
+    @Ignore
+    @Test
+    public void numberAttributesFacets() throws Exception {
+        final TermFacetExpression<ProductProjection, BigDecimal> termExpr = model().allVariants().attribute().ofNumber(ATTR_NAME_NUMBER)
+                .faceted().byAllTerms();
+        final ProductProjectionSearch termSearch = ProductProjectionSearch.ofStaged().withFacets(termExpr);
+        //assertThat(executeAndReturnTerms(termSearch, termExpr)).containsOnlyElementsOf(asList(TermStats.ofNumber(valueOf(5D), 1), TermStats.ofNumber(valueOf(10D), 1)));
```

Comment:
We, SDK. That's what I've been trying to fix for the last two days, with no success. I decided to leave it as it was and wait for the fixes from the API to have a new perspective.